recipes/hdf5/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX \
+    --enable-linux-lfs --with-zlib \
+    --with-pthread=yes  --enable-cxx --with-default-plugindir=$PREFIX/lib/hdf5/plugin \
+    --disable-silent-rules  # To make Travis happy with the level of activity.
```

Comment:
Here's the argument to add a bit more noise to the Travis CI build. Hopefully, that is the right amount without going overboard the other way.